CONTRIBUTING.md

```diff
@@ -25,8 +25,16 @@ guidance below.
 ## Changing the code-base
 
 Generally speaking, you should fork this repository, make changes in your
-own fork, and then submit a pull-request. All new code should have associated unit
```

Comment:
Interesting. Hasn't branching in cfgov-refresh been the standard for quite a while now? Does this signal an intention to go back to a fork-first model?